app/uk/gov/hmrc/valuationofficeagencycontactfrontend/handlers/ErrorHandler.scala

```diff
@@ -17,21 +17,23 @@
 package uk.gov.hmrc.valuationofficeagencycontactfrontend.handlers
 
 import javax.inject.{Inject, Singleton}
-
 import play.api.i18n.{I18nSupport, Messages, MessagesApi}
 import play.api.mvc.Request
 import play.twirl.api.Html
 import uk.gov.hmrc.valuationofficeagencycontactfrontend.FrontendAppConfig
 import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
+import uk.gov.hmrc.valuationofficeagencycontactfrontend.views.html.error_template
```

Comment:
*Nitpick (a k a feel free to ignore)*: It might be worth renaming the import to follow the CamelCase naming convention
```scala
import uk.gov.hmrc.valuationofficeagencycontactfrontend.views.html.{error_template => ErrorTemplate}
```

Reply:
I preffer not. It's html template. I can see from name that is template and not class.